docs/quickstart.md

```diff
@@ -101,17 +119,33 @@ class TinyNet:
     self.l2 = Linear(128, 10, bias=False)
 
   def __call__(self, x):
-    x = self.l1(x)
-    x = x.leakyrelu()
-    x = self.l2(x)
-    return x.log_softmax()
+    x = self.l1(x).leakyrelu()
+    return self.l2(x).log_softmax()
 
 net = TinyNet()
+
 ```
+
 We can see that the forward pass of our neural network is just the sequence of operations performed on the input tensor `x`.
-We can also see that functional operations like `leakyrelu` and `log_softmax` are not defined as classes and instead are just methods we can just call.
+We can also see that functional operations like `leakyrelu` and `log_softmax` are not defined as classes and instead are just methods we can just call like that or with `x = x.leakyrelu()`.
```

Comment:
This line becomes a little confusing with this change.

Reply:
yes, i'll rephrase it